WootricSDK/WootricSDK/UIItems.m

```diff
@@ -24,9 +24,28 @@
 
 #import "UIItems.h"
 #import "WTRColor.h"
+#import "NSString+FontAwesome.h"
+#import <CoreText/CTFontManager.h>
 
 @implementation UIItems
 
++ (void)dinamycallyAddFont {
+    NSString *fontPath = [[NSBundle bundleForClass:[UIItems class]] pathForResource:@"fontawesome-webfont" ofType:@"ttf"];
+    NSData *fontData = [NSData dataWithContentsOfFile:fontPath];
+    
+    CFErrorRef error;
+    CGDataProviderRef provider = CGDataProviderCreateWithCFData((CFDataRef) fontData);
+    CGFontRef font = CGFontCreateWithDataProvider(provider);
+    if (!CTFontManagerRegisterGraphicsFont(font, &error)) {
+        if (CFErrorGetCode(error) == kCTFontManagerErrorAlreadyRegistered) return;
```

Comment:
Use braces to avoid ambiguity.
`if (CFErrorGetCode(error) == kCTFontManagerErrorAlreadyRegistered) { return; }`